bsip-0058.md

```diff
@@ -67,17 +67,221 @@ related poll worker proposals, including FOR and AGAINST, will be created for vo
+
+
+##### <div id="gsp-activated-force-settling" /> Force-Settling
+
+Besides using the DEX to exchange a smart asset (e.g. bitCNY) to trade for its backing collateral (e.g. BTS), a holder of a smart asset has the ability "force-settle" (FS) a smart asset directly into the collateral subject to a delay of 24 hours and subject to a penalty that is specific to a smart asset (e.g. 5%).  This FS ability is true both when an asset is globally settled or not.  When it is not globally settled, the force settlement price of D units of a smart asset can be force settled for collateral equal to
+
+K = D &times; FSR &div; FP<sub>P</sub><sup>K/D</sup> = D &times; FSR &times; FP<sub>P</sub><sup>D/K</sup>
+
+where
+
+- D: amount of the smart asset that is held
+- FSR: force-settlement ratio (e.g. 95%)
+- FP<sub>P</sub><sup>K/D</sup>: published feed price 24 hours after the force settlement is initiated denominated in terms of smart asset divided by collateral (e.g. 2 bitCNY per BTS)
+- FP<sub>P</sub><sup>D/K</sup>: published feed price 24 hours after the force settlement is initiated denominated in terms of collateral divided by smart asset (e.g. 0.5 BTS per bitCNY)
+
+When a smart asset is globally settled, it is still possible to force settle with two differences in comparison to normal operations:
+
+- force-settlement has no delay; and,
+- the price used is the global settlement price (P<sub>gs</sub><sup>K/D</sup>), instead of using the feed price 24 hours in the future, which is denominated in units of the smart asset divided by the collateral (e.g. 2 bitCNY per BTS).  This price is **held constant** until the smart asset is revived from global settlement.
+
+K = D &times; FSR &div; P<sub>gs</sub><sup>K/D</sup> = D &times; FSR &times; P<sub>gs</sub><sup>D/K</sup>
+
+A GS-protected smart asset can still force-settled just as in normal operations albeit at an amount of collateral that is related to the GSP price.
+
+K = D &times; FSR &div; (1.01 &times; P<sub>gs</sub><sup>K/D</sup>) = D &times; FSR &times; (P<sub>gs</sub><sup>D/K</sup> &div; 1.01)
+
+_P<sub>gs</sub><sup>K/D</sup> is **not held constant** and will vary as the margin called debt positions are [purchased on the open market](#gsp-activated-trading)_.
+
+|Comparison to Globally Settled Asset|
+|-|
+|_While GSP is active_, a holder of a smart asset will obtain the __same amount or more collateral__ by force-settling than if the asset had globally settled because the GSP price will ratchet towards the natural feed price (FP<sub>N</sub>) as margin calls are filled.|
```

Comment:
IMHO better mention the delay here as well.

Reply:
@abitmore This sub-section has been re-arranged to highlight the delay here